js/everything.js

```diff
@@ -211,6 +217,30 @@ var ZenPlayer = {
                 that.show();
                 updateTweetMessage();
             });
+            // when player has finished playing
+            plyrPlayer.addEventListener("ended", function() {
+                videoMetaData = getParsedVideoMetaData();
+                if (videoMetaData) {
+                    videoMetaData["items"].push({"type" :"youtube", "id" :currentVideoID, "played" :true });
```

Comment:
I expect to see a space after the `:`, not before. Also, there's an extra space after `true`